### DISCUSSION TITLE
scaffolds.fasta deleted, can I regenerate it from SPAdes outputs?

### DISCUSSION THREAD

**Discussion by jamesck2** [ablab/spades#1076] [Q&A] [answered]
As the title suggests, I have accidentally deleted the scaffolds.fasta output for several assemblies. Fortunately, all other outputs from SPAdes are untouched. Is there a way I can regenerate the final scaffolds.fasta file from the existing outputs without re-running SPAdes in its entirety?

**Comment by asl** [comment, marked as answer]
If `output` is your output directory and 127 is the last k-mer length, then check for `output/K127/scaffolds.fasta` file.

**Reply by jamesck2** [reply]
Oh great! The K127 scaffolds are the same as the final scaffolds? If so that would solve my problem.

**Reply by asl** [reply]
Yes

**Reply by jamesck2** [reply]
Perfect, thank you for the help
